cmd/peggo/flags.go

```diff
@@ -40,6 +40,8 @@ const (
 	flagEthGasPrice          = "eth-gas-price"
 	flagEthGasLimit          = "eth-gas-limit"
 	flagPowerThreshold       = "power-threshold"
+	flagRelayerLoopDuration  = "relayer-loop-duration"
+	flagOrchLoopsDuration    = "orch-loops-duration"
```

Comment:
could we call this `loop` instead? wdyt?

Reply:
I used `loops` because it's used in 3 loops, but Idk, we can always change this later, it's not going to see much use besides tests